mobile/src/main/java/org/openhab/habdroid/ui/widget/SslClientCertificatePreference.java

```diff
@@ -21,6 +21,8 @@
 
 import java.security.cert.X509Certificate;
 
+import androidx.appcompat.widget.TooltipCompat;
```

Comment:
@maniac103 I'd to configure the import order three times, because of this issue: https://stackoverflow.com/questions/49295679/android-studio-3-0-settings-are-not-saved